config/routes.rb

```diff
@@ -8,4 +8,16 @@
       resources :likes, only: [:new, :create]
     end
   end
+
+  # ROUTES FOR API
+  namespace :api do
+    namespace :v1 do
+      post 'users/login' => 'users#login'
+      post 'users/signup' => 'users#signup'
```

Comment:
- [ ] Great work creating the routes for `login` and `signup` as suggested by the previous reviewer, however, your controller doesn't have the actions for login or signup, kindly make sure to implement them so your authentication is working properly on the API

![Screenshot_20221215_182301](https://user-images.githubusercontent.com/25227022/207926459-f33a7c16-e381-47c1-a3a1-765f9df8072f.png)

![Screenshot_20221215_181835](https://user-images.githubusercontent.com/25227022/207926296-044cc65c-250b-4688-b590-ca0def04379a.png)